src/FixedPointNumbers.jl

```diff
@@ -40,54 +41,87 @@ export
     # Functions
     scaledual
 
-reinterpret(x::FixedPoint) = x.i
+getindex(x::AbstractFixedPoint) = x.i
```

Comment:
This is interesting, but I worry it might give people the wrong idea that there's some kind of tuple or array container inside. So I'm on the fence about it. Care to provide a little explanation?